cirq/ops/common_channels_test.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+
+import cirq
+
+X = np.array([[0, 1], [1, 0]])
+Y = np.array( [[0, -1j], [1j, 0]])
+Z = np.array( [[1, 0], [0, -1]])
+
+
+def test_asymmetric_depolarizing_channel():
+    d = cirq.asymmetric_depolarize(0.1, 0.2, 0.3)
+    np.testing.assert_almost_equal(cirq.channel(d),
+                                   (np.sqrt(0.4) * np.eye(2),
+                                    np.sqrt(0.1) * X,
+                                    np.sqrt(0.2) * Y,
+                                    np.sqrt(0.3) * Z))
+
+
+def test_asymmetric_depolarizing_channel_repr():
+    cirq.testing.assert_equivalent_repr(
+        cirq.AsymmetricDepolarizingChannel(0.1, 0.2, 0.3))
+
+
+def test_asymmetric_depolarizing_channel_str():
+    assert (str(cirq.asymmetric_depolarize(0.1, 0.2, 0.3))
+            == 'AsymmetricDepolarizingChannel(p_x=0.1,p_y=0.2,p_z=0.3)')
+
+
+def test_asymmetric_depolarizing_channel_eq():
+    et = cirq.testing.EqualsTester()
+    et.add_equality_group(cirq.asymmetric_depolarize(0.0, 0.0, 0.0))
```

Comment:
This doesn't actually check that anything is ever equal to anything else. It would be satisfied by `def __eq__(a, b): return a is b`.